.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: [19.x, 18.x, 17.x, 16.x, 14.x, 12.x, "12.22.0"]
+        node: [20.x, 18.x, 16.x, 14.x, 12.x, "12.22.0"]
```

Comment:
I think we should not remove node.js v17/v19, as it's still supported as of eslint v8.

Reply:
Hmm, usually even number versions are stable. We can keep it. I don't have a strong opinion on this.

Reply:
```suggestion
        node: [20.x, 19.x, 18.x, 17.x, 16.x, 14.x, 12.x, "12.22.0"]
```